stdlib/concurrent/futures/process.pyi

```diff
@@ -10,6 +10,9 @@ from weakref import ref
 
 from ._base import Executor, Future
 
+_R = TypeVar("_R")
+_S = TypeVar("_S")
```

Comment:
Any reason not to use a single TypeVar named `_T`?

Reply:
Done